### PR TITLE
Update CHARTER.md

### PR DESCRIPTION
**Pull request by chris-short**
Refreshing post name change

**Which issue(s) this PR fixes**:

Fixes # N/A

While creating the Subproject Lead Handbook, @kaslin and I realized the CHARTER.md was out of date and improvements could be made.

We will ping the Slack channel this week and bring it up in our meeting next week. Lazy concensus will begin after meeting.

### CODE REVIEW

**Comment by chris-short** [comment]
/assign @kaslin 

/hold

Waiting for next meeting to discuss changes.

File: communication/contributor-comms/CHARTER.md [hunk 1]
```diff
@@ -18,22 +18,34 @@ This team operates independently from Cloud Native Computing Foundation (CNCF) o
...
 
-Roles are divided up between Marketing Council members and other members, notably Storytellers. Marketing Council members are scoped to have at least one shadow. All leads should actively recruit shadows for the entire team.
+General subproject leadership is handled by the Contributor Comms Subproject Lead(s). Specific subproject responsibilities are also broken down into areas with their own lead roles, as defined here:
+
+* Contributor Comms Subproject Lead
+* [Social Media Coordinator](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/Social-Media.md)
+* [Comms Tech Lead](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/Comms-Tech-Lead.md)
+* [Comms Event Lead](https://github.com/chris-short/community/blob/master/communication/contributor-comms/role-handbooks/Comms-Event-Lead.md)
+* [Comms Blog Coordinator](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/blog-coordinator.md)
+* [Storytellers](https://github.com/chris-short/community/blob/master/communication/contributor-comms/role-handbooks/storytellers.md) & other members
```
Comment (jberkus) [inline communication/contributor-comms/CHARTER.md:32]: Bad links

File: communication/contributor-comms/CHARTER.md [hunk 2]
```diff
@@ -18,22 +18,34 @@ This team operates independently from Cloud Native Computing Foundation (CNCF) o
 
 ## Governance
 
-Leadership is divided up between [Marketing Council] team members. This group of leaders divide up the core responsibilities of the team and look to further optimize the experience of both code and non-code contributors.
+The Contributor Comms subproject welcomes and encourages its members to take on leadership roles in their contributions. The group defines the areas of its work such as social media, blogging, etc. and creates associated leadership roles accordingly. Members are encouraged to pursue any interest in specific areas as contributing team members, leads, or shadows. Leadership roles are defined in the subproject’s [role handbooks](https://github.com/chris-short/community/tree/master/communication/contributor-comms/role-handbooks).
```
Comment (jberkus) [inline communication/contributor-comms/CHARTER.md:21]: bad link

File: communication/contributor-comms/CHARTER.md [hunk 3]
```diff
@@ -18,22 +18,34 @@ This team operates independently from Cloud Native Computing Foundation (CNCF) o
...
+Decisions affecting all of Contributor Comms are made primarily led by the Contributor Comms Subproject Lead[s]. Decisions are generally made by lazy consensus with the goal of everyone getting a say, but leads are ultimately responsible for making the decision. If team members do not agree with one or more decisions made on behalf of the team, this should be raised to the subproject lead[s] as well as the SIG Contributor Experience leads to facilitate a resolution.
 
-Other council leads have the responsibility and autonomy to decide what is right for their portion of the council's division of work. The same escalation path can be used in case of disagreement that cannot be resolved by internal discussion.
+Leads have the responsibility and autonomy to decide what is right for their portion of the groupl's division of work. The same escalation path can be used in case of disagreement that cannot be resolved by internal discussion.
```
Comment (jberkus) [inline communication/contributor-comms/CHARTER.md:45]: typo "group"

File: communication/contributor-comms/CHARTER.md [hunk 4]
```diff
@@ -18,22 +18,34 @@ This team operates independently from Cloud Native Computing Foundation (CNCF) o
...
+
+Roles are flexible and subject to change.
+
+Roles are occupied or available as indicated on our [README.md](https://github.com/kubernetes/community#readme).
 
 ### Decision Making
 
-Decisions affecting all of Contributor Comms are made primarily led by the program lead as defined in the [Marketing Council]. They come to decision by lazy consensus with the goal of everyone getting a say, but they are responsible for making the decision. If other council members do not agree with one or more decisions made on behalf of the team, this should be raised to the advisors as well as the SIG Contributor Experience leads to facilitate a resolution.
+Decisions affecting all of Contributor Comms are made primarily led by the Contributor Comms Subproject Lead[s]. Decisions are generally made by lazy consensus with the goal of everyone getting a say, but leads are ultimately responsible for making the decision. If team members do not agree with one or more decisions made on behalf of the team, this should be raised to the subproject lead[s] as well as the SIG Contributor Experience leads to facilitate a resolution.
```
Comment (jberkus) [inline communication/contributor-comms/CHARTER.md:43]: lead(s)

**Review by jberkus — CHANGES_REQUESTED** [review]
A few typos, plus all of the new links you added are absolute links against your fork of the repository.  Please use relative links where possible, and where not possible link the master branch of the main repo.  Thanks!

**Comment by chris-short** [comment]
@jberkus Noted. Lost my local copy. Will open a new PR shortly.

**Pull request closed without merging**